spec/lib/efile/nj/nj1040_spec.rb

```diff
@@ -177,12 +177,8 @@ def over_65_birth_year
   describe 'line 39 - taxable income' do
     let(:intake) { create(:state_file_nj_intake, :df_data_2_w2s, :primary_over_65, :primary_blind) }
     it 'sets line 39 to line 29 gross income minus line 38 total exemptions/deductions' do
-      line_15_w2_wages = (12345.67 + 50000).round
-      self_exemption = 1_000
-      self_over_65 = 1_000
-      self_blind = 1_000
-      total_exemptions = self_exemption + self_over_65 + self_blind
-      expect(instance.lines[:NJ1040_LINE_39].value).to eq(line_15_w2_wages - total_exemptions)
+      expected_total = instance.lines[:NJ1040_LINE_29].value - instance.lines[:NJ1040_LINE_38].value
+      expect(instance.lines[:NJ1040_LINE_39].value).to eq(expected_total)
```

Comment:
it looks like this is failing?

Reply:
@tahsinaislam I squashed my commits and re-ran the tests, all green now. Looks like it was just a flaky test. Going to go ahead and merge